docs/get_started/quickstart.mdx

```diff
@@ -532,6 +532,136 @@ go build .
 
 </details>
 
+</TabItem>
+<TabItem value="python" label="Python" default>
+
+        <Admonition type="note" title="Prerequisites">
+            - Python >= v3.11
+            - Optional but recommended: [Install the Restate Server & CLI](/develop/local_dev#running-restate-server--cli-locally)
+        </Admonition>
+
+        <Step stepLabel="1" title="Get the Greeter service template">
+
+            <CH.Code>
+                ```shell CLI
+                # link(1:3) /develop/local_dev#running-restate-server--cli-locally
+                restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell npx
+                npx @restatedev/restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell wget
+                wget https://github.com/restatedev/examples/releases/latest/download/python-hello-world.zip &&
+                unzip python-hello-world.zip -d python-hello-world &&
+                rm python-hello-world.zip && cd python-hello-world
+                ```
+
+            </CH.Code>
+
+        </Step>
+        <Step stepLabel="2" title="Run the Greeter service">
+
+            Create a `venv` and install the requirements:
+
+            ```shell
+            python3 -m venv .venv
+            source .venv/bin/activate
+            pip install -r requirements.txt
+            ```
+
+            Now, start developing your service in `example.py`. Run it with a [Hypercorn](https://pypi.org/project/Hypercorn/) server, and let it listen on port `9080` for requests:
+
+            ```shell
+            python -m hypercorn --config hypercorn-config.toml -b localhost:9080 example:app
+            ```
+
+        </Step>
+
+        <Step stepLabel="3" title="Launch Restate">
+            Restate is a single self-contained binary. No external dependencies needed. Run it locally via ([or download the binaries](/develop/local_dev#running-restate-server--cli-locally)):
+
+
+            <CH.Code rows={"2"}>
+
+                ```shell npx
```

Comment:
Same comment as before, in particular here it's the first choice!

Reply:
Actually this comment applies to the other languages too, in java, golang, i get npx as first choice. it's ok if you wanna keep it, but it shouldn't be the first choice!

Reply:
So do I then link to the download script for the binary or what should I put first? Cause I also don't like Docker as first option. And Homebrew is not general enough.

Reply:
download script for the binary is fine, but also docker i think it's fine.

Reply:
But definitely IMO it's not fine that npx is the first choice for anything other than typescript